src/clr-angular/forms/common/form.ts

```diff
@@ -28,4 +52,71 @@ export class ClrForm {
   markAsTouched() {
     this.markControlService.markAsTouched();
   }
+
+  @ContentChildren(ClrLabel, { descendants: true })
+  labels: QueryList<ClrLabel>;
+
+  @HostListener('submit')
+  onFormSubmit() {
+    this.markAsTouched();
+    if (isPlatformBrowser(this.platformId)) {
+      this.invalidControls = Array.from(this.el.nativeElement.querySelectorAll('.ng-invalid'));
+      if (this.invalidControls.length > 0) {
+        this.invalidControls[0].focus();
+        this.updateAriaLive();
+      } else {
+        this.resetAriaLive();
+      }
+    }
+  }
+
+  ngOnInit() {
+    this.createAriaLiveElement();
+  }
+
+  private resetAriaLive() {
+    this.ariaLiveElement.innerHTML = '';
+  }
+
+  private updateAriaLive(): void {
+    if (this.invalidControls.length === 0) {
+      return;
+    }
+
+    this.ariaLiveText = this.commonStrings.keys.formErrorSummary;
```

Comment:
Is this `this.ariaLiveText` being used anywhere?

Reply:
nope its left over from the other iterations. Will remove.